README.md

```diff
@@ -132,7 +132,7 @@ Example: Train.jsonl
 
 ## Supported Models
 
-Current supported and tested models are `Llama2` (7 and 13B configurations have been tested) and `GPTBigCode`.
+Current supported and tested models are `Llama3` (8B configuration has been tested) and `GPTBigCode`.
```

Comment:
We actually support many more models than this. @aluu317 had started working on adding a matrix of the models we've tested and supported. This would include mistral, allam, granite with llama arch, and granite GraniteForCausalLM models

Reply:
@aluu317 Want me to change to something like the below? I can update the table if you give me the list of tested models or feel free to make the changes yourself too
| | LoRA | PR | FT | GPTQ-LoRA |
|---|------|-----|-----|-----|
| Granite with LLama architecture |✅|---|---|---|
| Granite (GraniteForCausalLM)   |✅|---|---|---|
| Mistral |---|---|---|---|
| Allam |---|---|---|---|